### ISSUE TITLE
Newsletter June 2022

### ISSUE THREAD

**Issue by ahmadabugosh** [Giveth/giveth-planning#723]

**Comment by ahmadabugosh** [comment]
@laurenluz added this issue to dump anything that comes up for june

**Comment by laurenluz** [comment]
Great! Thanks Ahmad! I will add to it as we go. For now we should definitely include:

- hitchhiker's guide blog post
- CULT / ETH regenfarm

**Comment by laurenluz** [comment]
I gave an awesome talk at metafest about Giveth, we should include that too

**Comment by laurenluz** [comment]
I think the featured project should be bob burnquest's since he was on the community call and it was super hyped

https://twitter.com/WillyOgo/status/1537190867529658368

**Comment by laurenluz** [comment]
should also feature Griff @ consensus. he gave an aweomse talk on the biggest web3 opportunity... I'll see if I can find a recording

**Comment by laurenluz** [comment]
https://www.youtube.com/watch?v=NDGeb-bNH3I Here is the link to the community call w/ bob burnquist... I'm sorry it got cut off at the end :(

**Comment by ahmadabugosh** [comment]
Hey @laurenluz @danelsuga @c0ric0ri , I made a draft for the June newsletter here: https://docs.google.com/document/d/1p4h1y8YPzxwh2P85VuaNutaUTl4CIWYv6LIyZahwchs/edit. 

Even though I'm off, I plan to send it out by this Thursday, so please let me know if you have any comments (I also left some comments on the sheet I could use input on) 🙏

**Comment by danelsuga** [comment]
Great! Thanks, Ahmad! Looking good. Left a couple little comments.

**Comment by laurenluz** [comment]
Great job @ahmadabugosh, as usual. Left some suggestions in the doc. I'll try to get that consensus talk video approved if I can 😕

**Comment by laurenluz** [comment]
They sent me a link! Griff's part starts at 1hr: https://www.coindesk.com/video/recent-videos/consensus-2022-foundations-ethereum/

@ahmadabugosh

**Comment by ahmadabugosh** [comment]
Thanks for the support @laurenluz & @danelsuga !

@laurenluz for the auto tweet message, are you fine with this?
" It may seem like a slow crypto month, but we've been keeping busy at Giveth. Read our top GIVnews highlights from June:
🗺 Hitchhiker’s Guide to the Giveth Galaxy 
🤑 New GIV/DAI Farm 
👾 [@karmaticacid](https://twitter.com/karmaticacid)
 at [@MetaFam](https://twitter.com/MetaFam)
👥 [@thegrifft](https://twitter.com/thegrifft)
 at [@consensus2022](https://twitter.com/consensus2022)
  and more!"

**Comment by laurenluz** [comment]
Yeah that's fine! Will it go out tomorrow?

**Comment by ahmadabugosh** [comment]
@laurenluz Yes, was planning on it! Let me know if you have a time preference (should I make it 8:40am costa rica time like last time)?

**Comment by laurenluz** [comment]
Yeah that's great! Thanks @ahmadabugosh
